src/Controller/SsoController.php

```diff
@@ -9,6 +9,11 @@
 
 class SsoController extends BaseController
 {
+    public function error($response, $message, $statusCode){
```

Comment:
Also, these parameters should be typehinted.
`Response`
`string`
`integer`

PHPStorm will fix this automatically for you when you hover over and click on these in the IDE.

Reply:
Should probably default to error code 500

Reply:
I added types and error code now @Lundli